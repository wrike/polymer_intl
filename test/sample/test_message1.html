<html>
  <head>
  	<meta charset="utf-8">
  	<meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>test1</title>
  </head>

  <body>
    <p id="text"></p>
    <i18n-message>
      <value>The protection \$arg1 of property rights $arg2 remains one of the most $arg3 contentious issues in $arg1 present-day Russia \$arg1.</value>
      <meaning>meaning - Lorem ipsum dolor sit amet, consectetur adipisicing </meaning>
      <arg id="arg1" example="test1">1</arg>
      <arg id="arg2" example="test2">2</arg>
      <arg id="arg3" example="test3">3</arg>
      <desc>desc - Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia, neque quidem officiis sit perferendis</desc>
    </i18n-message>

    <i18n-message locale="EN">
      <value>2The protection \$arg1 of property rights $arg2 remains one of the most $arg3 contentious issues in $arg1 present-day Russia \$arg1.</value>
      <meaning>2meaning - Lorem ipsum dolor sit amet, consectetur adipisicing </meaning>
      <arg id="arg1" example="test1">{{value1}}</arg>
      <arg id="arg2" example="test2">2</arg>
      <arg id="arg3" example="test3">{{value2}}</arg>
      <desc>2desc - Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia, neque quidem officiis sit perferendis</desc>
    </i18n-message>

    <i18n-message>
      <value>Msg3 Only Text</value>
      <meaning>Msg3 meaning - Lorem ipsum dolor sit amet, consectetur adipisicing </meaning>
      <desc>3desc - Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia, neque quidem officiis sit perferendis</desc>
    </i18n-message>

    <i18n-gender>
      <value>4The protection \$arg1 of property rights $arg2 remains one of the most \$arg3 contentious issues in $arg1 present-day Russia \$arg1.</value>
      <meaning>4meaning - Lorem ipsum dolor sit amet, consectetur adipisicing </meaning>
      <male>male</male>
      <female>female</female>
      <default>other</default>
      <arg id="arg1" example="test1">{{value1}}</arg>
      <arg id="arg2" example="test2">2</arg>
      <desc>4desc - Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia, neque quidem officiis sit perferendis</desc>
    </i18n-gender>

    <i18n-plural>
      <zero>zero</zero>
      <one>one</one>
      <two>two</two>
      <few>few</few>
      <many>many</many>
      <default>other</default>
      <value>3</value>
      <meaning>5meaning - Lorem ipsum dolor sit amet, consectetur adipisicing </meaning>
      <arg id="arg1" example="test1">{{value1}}</arg>
      <arg id="arg2" example="test2">2</arg>
      <desc>5desc - Lorem ipsum dolor sit amet, consectetur adipisicing elit. Mollitia, neque quidem officiis sit perferendis</desc>
    </i18n-plural>
  </body>
</html>