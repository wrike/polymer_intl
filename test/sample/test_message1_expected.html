<html>
  <head>
  	<meta charset="utf-8">
  	<meta name="viewport" content="width=device-width, initial-scale=1.0">
    <title>test1</title>
  </head>
 
  <body>
    <p id="text"></p>
    {{message_msg1("1", "2", "3")}}
    
    {{message_msg2_with_param(value1, "2", value2)}}
    
    {{message_msg3_without_args()}}
    
    {{message_msg4_without_args()}}
  </body>
</html>
